.travis.yml

```diff
@@ -45,7 +50,9 @@ deploy:
     local_dir: build
     region: $AWS_REGION
     on:
-      branch: master
+      # branch: master
+      all_branches: true
+      condition: $TRAVIS_BRANCH =~ ^(master|feature/xdai-deployment)$
```

Comment:
here `master` is on the same level as `feature/xdai-deployment` ... in this case I would recommend not to call it `feature/...` (maybe just `xdai-deployment` or `release/...` not sure). Feature branches are for me used when developing a feature.

Reply:
I think it's just temporal, then all the environments will be deployed together when we merge to master.

Reply:
@rmeissner updated. I think `xdai-deployment` it's self explanatory

Reply:
Yes, this was a special case to make it deploy to staging without merging to master. I think that this can be removed now